_data/software/demeter.yml

```diff
@@ -0,0 +1,36 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: >-
+      Demeter is a comprehensive system for processing and analyzing X-ray
+      Absorption Spectroscopy data.
+    fr: >-
+      Demeter est un système complet de traitement et d'analyse des données de
+      spectroscopie d'absorption des rayons X.
+category: '43232600'
+homepageURL:
+  en: 'https://bruceravel.github.io/demeter/'
+  fr: 'https://bruceravel.github.io/demeter/'
+licences:
+  - URL:
+      en: 'https://github.com/bruceravel/demeter/blob/master/LICENSE'
+      fr: 'https://github.com/bruceravel/demeter/blob/master/LICENSE'
+    spdxID: Artistic-2.0
```

Comment:
The text in license file is Artistic, but the README talk about GPL https://github.com/bruceravel/demeter/#licence-and-copyright https://metacpan.org/pod/perlgpl

Reply:
It's an OR conditional for GPL1.0 or the Artistic, we can't accept GPL1.0, so Artistic it is.